docs/document/content/user-manual/shardingsphere-proxy/distsql/_index.cn.md

```diff
@@ -4,4 +4,31 @@ weight = 3
 chapter = true
 +++
 
-本章节将介绍 [DistSQL](/cn/concepts/distsql/) 的详细语法。
+本章节将介绍 [DistSQL](/cn/overview/distsql/) 的详细语法。
+
+## 语法规则
+
+在 DistSQL 语句中，除关键字外，其余元素的输入格式应符合以下规则。
+
+### 标识符
+
+1. 标识符代表 SQL 语句中的一个对象，包括：
+
+- 数据库名称
+- 表名
+- 列名
+- 索引名称
+- 资源名称
+- 规则名称
+- 算法名称
+
+1. 标识符中允许使用的字符有：[a-z,A-Z,0-9,_]（字母、数字、下划线），且应以字母开头。
+1. 当标识符中出现关键字或特殊字符时，使用反引号 (`) 。
+### 字面量
```

Comment:
Please leave a blank line between new title and last content.

Reply:
have added a blank line.